java/vector/src/main/codegen/templates/AbstractFieldWriter.java

```diff
@@ -98,6 +98,12 @@ public ListWriter list() {
   }
 
   @Override
+  public ListWriter list(int size) {
+    fail("FixedSizeList");
+    return null;
+  }
```

Comment:
- you can write to the vector directly through its mutator. The Writer api is a facility to write in a row-oriented fashion to the vectors.
- How does this interact with the PromotableWriter? what happens if we write both fixed lists and variable size lists to the same field?